### PR TITLE
Simplify product dirs

### PR DESCRIPTION
**Pull request by mpreisler**
We have done a lot towards making SSG easier to contribute to. Despite that we recently discovered that creating a new product directory structure is still a really daunting task. I wanted to improve the situation by simplifying the dir structure as much as I could but it seems I am removing too much and I may be wrong to do that. A lot of the XSLTs in $PRODUCT/transforms are completely unused, not referenced in any docs, not used in the build system, not included in any other XSLT.

Should I just remove them all? Is there any reason why they are there?

### CODE REVIEW

**Comment by redhatrises** [comment]
@mpreisler this was most likely a product of the old make system when you used to be able to `cd <product> && make`. However, having the transforms in the local product directories also allows for being able to override any shared setting made in the shared XSLT.

File: shared/modules/xccdf2csv_stig_module.py [hunk 1]
```diff
@@ -1,58 +0,0 @@
-#!/usr/bin/python2
```
Comment (redhatrises) [inline shared/modules/xccdf2csv_stig_module.py]: Not sure. But I could see this file still be useful.

Comment (mpreisler) [inline shared/modules/xccdf2csv_stig_module.py]: I have removed this commit because we are not sure.

**Comment by jan-cerny** [comment]
Thank you :D

**Pull request merged**